meta-resin-common/classes/image_types_resin.bbclass

```diff
@@ -258,3 +260,9 @@ IMAGE_CMD_resinos-img () {
 # Make sure we regenerate images if we modify the files that go in the boot
 # partition
 do_rootfs[vardeps] += "RESIN_BOOT_PARTITION_FILES"
+
+# XXX(petrosagg): This should be eventually implemented using a docker-native daemon
+IMAGE_CMD_docker () {
+    DOCKER_IMAGE=$(${IMAGE_CMD_TAR} -cv -C ${IMAGE_ROOTFS} . | DOCKER_API_VERSION=1.22 docker import -)
```

Comment:
There are a set of references now to docker api 1.22. It would be nice to have a variable for this or detect the version before using it.

Reply:
Setting the API version to something older isn't a problem. Newer docker daemons will handle it just fine. Also, this will go away once we move our daemon binary to another name so that it doesn't conflict with the host docker